python/nano/scripts/bigdl-nano-init

```diff
@@ -300,10 +300,11 @@ if [ "${OPENMP}" -eq 1 ]; then
         let threads_per_core=$cpu_/$core_
         let cores_per_socket=$core_/$sockets_
         export OMP_NUM_THREADS=$core_
-        export MKL_NUM_THREADS=$core_
+        unset MKL_NUM_THREADS  # to avoid inc issue when this var is set diff with OMP_NUM_THREADS
```

Comment:
Maybe you should add it outside this if branch.

Btw, you should unset it, `OMP_PROC_BIND` and `OMP_SCHEDULE`  in `bigdl-nano-unset-nev`

Reply:
yeah good point, will fix